README.md

```diff
@@ -47,8 +47,21 @@ Install pip for python3
 1. Login the admin panel and create entries in the Navbar Entries section as per your requirement.
 2. Load and use the `navbar` tag if required in any of your custom pages.
 
+### Setting up email templates
+1. Create templates in both text and html formats.
+2. Place them in the directory cited below.
```

Comment:
These could be called in a way similar to [this](https://github.com/nkprince007/festcms/blob/mail-extras/festflow/views.py#L132,#L135)